api/observability/v1/output_types.go

```diff
@@ -593,12 +593,18 @@ type HTTP struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Timeout",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:number"}
 	Timeout int `json:"timeout,omitempty"`
 
-	// Method specifies the Http method to be used for sending logs. If not set, 'POST' is used.
+	// Method specifies the HTTP method to be used for sending logs. If not set, 'POST' is used.
 	//
 	// +kubebuilder:validation:Enum:=GET;HEAD;POST;PUT;DELETE;OPTIONS;TRACE;PATCH
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="HTTP Method"
 	Method string `json:"method,omitempty"`
+
+	// ProxyURL if specified is used to proxy HTTP and HTTPS traffic.
```

Comment:
@alanconway do we need to explicitly state that SOCKS proxy is not supported?

Reply:
I don't think so, but I made the wording more explict to say we're talking about a HTTP proxy, not  a general socket proxy:

	// ProxyURL URL of a HTTP or HTTPS proxy to be used instead of direct connection.